### PR TITLE
Invitation lock tooltip (#3479)

### LINKED ISSUE #3479: Tooltip for Invitation Lock
## Scope

- Add tooltip to invitation lock

"Invitation lock can be spent on transaction fees and staking for proposals, voting and working groups applications. JOY tokens subject to this lock cannot be transferred to any other accounts. This lock is unrecoverable. NB: Transaction fees will first be taken from your transferable balance if it is positive."

Learn more about locks>: https://joystream.gitbook.io/testnet-workspace/system/accounts-and-staking"

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201958687417145/1202809828964348) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by traumschule**
Fixes #3479

Note: Is this the right spot? Took the liberty to also address `Creation fee`, feel free to expand.

### CODE REVIEW

**Comment by WRadoslaw** [comment]
@dmtrjsg I don't see any issue for this tooltip change, please feel free to take a look and approve if everything is good

File: packages/ui/src/memberships/modals/BuyMembershipModal/BuyMembershipSignModal.tsx [hunk 1]
```diff
@@ -97,8 +97,10 @@ export const BuyMembershipSignModal = ({
           <Row>
             <Info>
               <TextMedium>
-                Tokens subject to Vesting and Invitation locks do not cover membership creation fees, and cannot be used
-                to purchase new memberships.
+                Invitation lock can be spent on transaction fees and staking for proposals, voting and working groups
```
Comment (traumschule) [inline packages/ui/src/memberships/modals/BuyMembershipModal/BuyMembershipSignModal.tsx:100]: In light of #3321 i'm wondering if it should be more clear like `can ONLY be spent on`
> show some very clear big visual that sort of says "you will get some tiny amount of free coins, but you can only use them for fees, not transfer or selling, that wont work"

**Comment by dmtrjsg** [comment]
LGTM!

**Review by WRadoslaw — APPROVED** [review]

File: packages/ui/src/memberships/modals/BuyMembershipModal/BuyMembershipSignModal.tsx [hunk 2]
```diff
@@ -111,7 +113,7 @@ export const BuyMembershipSignModal = ({
         <TransactionInfo
           title="Creation fee:"
           value={membershipPrice?.toBn()}
-          tooltipText={'Lorem ipsum dolor sit amet consectetur, adipisicing elit.'}
+          tooltipText={'The price to create a membership.'}
```
[the remark this answers is not in the document]
Comment (traumschule) [reply packages/ui/src/memberships/modals/BuyMembershipModal/BuyMembershipSignModal.tsx:116]: Fixed in #3775.

**Pull request merged**